cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -141,6 +145,30 @@ func (o *AddOrUpdateOptions) Run(args []string) error {
 		return err
 	}
 
+	if o.CreateNamespace {
+		o.statusUI.PrintMessagef("Creating namespace '%s'", o.NamespaceFlags.Name)
```

Comment:
Should we remove this line as well? Or you already did and forgot to push the change?